FunctionHomework/src/TheTime/DataValidation.java

```diff
@@ -0,0 +1,24 @@
+package TheTime;
+
+import java.util.Scanner;
+
+public class DataValidation {
+    public static void main(String []args ) {
+        Scanner s = new Scanner(System.in);
+        System.out.println("What time is it? ");
+        System.out.println("Enter the hour: ");
+        int h = s.nextInt();
+        System.out.println("Enter the minutes: ");
+        int m = s.nextInt();
+        if (valid(h , m)) {
+            System.out.println("The time is  " + h + ":" + m + " now." );
```

Comment:
you should have done something with processing to string to display 02:03 instead of 2:3